pkg/clusters/addons/kong/kong.go

```diff
@@ -146,6 +153,15 @@ func (a *Addon) Deploy(ctx context.Context, cluster clusters.Cluster) error {
 		)
 	}
 
+	imageRepo := fmt.Sprintf("image.repository=%s", EnterpriseImageRepo)
+	imageTag := fmt.Sprintf("image.tag=%s", EnterpriseImageTag)
+	if a.enterperise {
+		a.deployArgs = append(a.deployArgs,
+			"--set", imageRepo,
+			"--set", imageTag,
+		)
+	}
+
```

Comment:
This doesn't yet take into account user provided `image` or `tag` and we should make sure those are provided, and these images are the backup defaults.

Reply:
eeff8d91ce826dd138626da30da6711a030c8649